src/server/game/Texts/ChatTextBuilder.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license, you may redistribute it and/or modify it under version 2 of the License, or (at your option), any later version.
```

Comment:
let's always use AGPL for new files. It is compatible with "GPLv2 or later" from TC